Split/FetcherEngine/Refresh/LocalhostSplitFetcher.swift

```diff
@@ -67,35 +70,23 @@ class LocalhostTreatmentFetcher: TreatmentFetcher {
     }
     
     private func loadFile() {
-        if let content = fileStorage.read(fileName: fullFileName), !content.isEmpty {
+        if let content = fileStorage.read(fileName: fileName), !content.isEmpty {
             let loadedSplits = fileParser.parseContent(content)
             splits.setValues(loadedSplits)
         }
     }
     
-    private func initFile() {
-        var fileContent: String? = nil
-        
-        if let filepath = Bundle.main.path(forResource: fileName, ofType: fileType) {
-            do {
-                fileContent = try String(contentsOfFile: filepath, encoding: .utf8)
-            } catch {
-                Logger.e("File Read Error for file \(filepath)")
-            }
-        }
-        if fileContent == nil {
-            Logger.w("Localhost file or content not found. An empty file will be created")
+    private func getParser() -> LocalhostSplitsParser {
+        if fileName.contains(".yaml") || fileName.contains(".yml") {
```

Comment:
Wouldn't endsWith be a better and safer check? myWrong.ymlFile.xls shouldn't be parsed as yaml (should not be parsed at all)

Reply:
Thanks for the suggestion @NicoZelaya. This Check was replaced by a much more effective one.